crates/shared/src/types.rs

```diff
@@ -105,6 +105,8 @@ pub enum HashingAlgorithm {
     Sha3,
     Keccak,
     Blake2_256,
+    /// If you are trying to sign a hashed message you can use this, length requirement is 64 digits
+    NoHash,
```

Comment:
```suggestion
    Identity,
```

This is also the terminology used by Substrate for a hashing function which produces the same output as its input

Reply:
yes works but Im just gonna be pedantic tho, cuz doesn't that mean it is not a hashing algorithim if it has the same output lol